tests/test_api.py

```diff
@@ -113,3 +123,10 @@ def test_continue_trace(sentry_init):
         assert propagation_context["dynamic_sampling_context"] == {
             "trace_id": "566e3688a61d4bc888951642d6f14a19"
         }
+
+
+@pytest.mark.skipif(REDIS_INSTALLED, reason="skipping because redis is installed")
+def test_redis_disabled_when_not_installed(sentry_init):
+    sentry_init()
+
+    assert Hub.current.get_integration(RedisIntegration) is None
```

Comment:
I was unsure about in which file this test should be placed. Is it ok to keep here, or should it go somewhere else?

Reply:
Maybe `test_basics.py` would be better? It already has some tests for integration enabling.

`test_api.py` is more for testing the global API (`configure_scope`, `start_transaction`, etc.; the stuff we surface via https://github.com/getsentry/sentry-python/blob/cb7299ad062f9c7c7aa66306816915e892e9e040/sentry_sdk/__init__.py#L13-L43).